ixwebsocket/IXWebSocket.h

```diff
@@ -152,9 +158,15 @@ namespace ix
         std::atomic<int> _handshakeTimeoutSecs;
         static const int kDefaultHandShakeTimeoutSecs;
 
-        // Optional Heartbeat
-        int _heartBeatPeriod;
-        static const int kDefaultHeartBeatPeriod;
+        // enable or disable PONG frame response to received PING frame
+        bool _enablePong;
+        static const bool kDefaultEnablePong;
```

Comment:
I prefer to assign constant values directly in header.

Reply:
@bsergean is always doing that in the body, so I keep the same way